Scrape-Subreddit-Users-master/GetSubredditSubscribersComments.py

```diff
@@ -35,16 +36,15 @@
             pass
         else:
             session.add(User(username))
+            session.commit()
```

Comment:
There just isn't really a reason to hold off on committing. Might as well commit as you go -- you'll save more data if you hit an error.